services/workflows-service/src/workflow/schemas/extenstions.schemas.ts

```diff
@@ -0,0 +1,87 @@
+import { Type, Static } from '@sinclair/typebox';
+
+const TransformSchema = Type.Object({
+  mapping: Type.Union([
+    Type.String(),
+    Type.Array(
+      Type.Object({
+        method: Type.String(),
+        source: Type.String(),
+        target: Type.String(),
+      }),
+    ),
+  ]),
+  transformer: Type.String(),
+});
+
+const ApiPluginSchema = Type.Object({
```

Comment:
General comment about the plugins schema - some of the inner properties (like `request` or `response`) are not required. It could break the schema validation if there's a plugin that doesn't have one of these properties. Please make sure what's required and what's optional and add `Type.Optional()` accordingly

Reply:
sure